components/crunchbase/actions/search-organizations/search-organizations.mjs

```diff
@@ -0,0 +1,37 @@
+import { FIELDS_OPTIONS } from "../../common/constants.mjs";
+import { parseObject } from "../../common/utils.mjs";
+import crunchbase from "../../crunchbase.app.mjs";
+
+export default {
+  key: "crunchbase-search-organizations",
+  name: "Search Organizations",
+  description: "Search for organizations based on specified criteria. [See the documentation](https://data.crunchbase.com/reference/post_data-searches-organizations)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    crunchbase,
+    fieldIds: {
+      type: "string[]",
+      label: "Field IDs",
+      description: "Fields to include on the resulting entity",
+      options: FIELDS_OPTIONS,
+    },
+    query: {
+      type: "string[]",
+      label: "Query",
+      description: "Array of stringified objects for searching organizations",
+      optional: true,
+    },
```

Comment:
I think the `query` prop might be confusing for some users. What if instead of the `query` prop, there were `fieldId`, `operatorId`, `type`, and `values` props. There would only be one query object allowed in the request, but we could create it for the user based on those other props. What do you think?

Reply:
I think only one object is not good. I'll add an example and a link on the description. I think it's better.